### ISSUE TITLE
想请教一下关于BatchNorm的问题

### ISSUE THREAD

**Issue by luwanglin** [ShiqiYu/OpenGait#19]
我在代码中看到，很多网络中都没有加上BatchNorm，效果和论文中差不多甚至更高，但是我实验了一下，加上BatchNorm后反而效果变差了，不知道这个原因是啥呢？

**Comment by zhangzhi-git** [comment]
而且我尝试过加入注意力貌似没有用。

**Comment by noahshen98** [comment]
I would recommand paper for reference:https://ieeexplore.ieee.org/abstract/document/9410587
seem this work integrated bn within each block.

The reason that causing performance degradation maight be whether synchronized version of batch normalization in used. see https://pytorch.org/docs/stable/generated/torch.nn.SyncBatchNorm.html.

But I am not sure because I haven't tried it yet. it just my personal thought.

**Comment by luwanglin** [comment]
> I would recommand paper for reference:https://ieeexplore.ieee.org/abstract/document/9410587 seem this work integrated bn within each block.
> 
> The reason that causing performance degradation maight be whether synchronized version of batch normalization in used. see https://pytorch.org/docs/stable/generated/torch.nn.SyncBatchNorm.html.
> 
> But I am not sure because I haven't tried it yet. it just my personal thought.

感谢您的解答

**Comment by luwanglin** [comment]
> 而且我尝试过加入注意力貌似没有用。

的确，我之前尝试过加入一些注意力模块，但是效果没有变好反而更差了
